scripts/parse_ped.py

```diff
@@ -0,0 +1,40 @@
+""" A really simple script to import a pedigree file """
+
+import click
+from google.cloud import storage
+
+from sample_metadata.apis import (
+    FamilyApi,
+)
+
+
+@click.command()
+@click.option('--ped-file-path', 'ped_file_path')
+@click.option('--bucket', 'bucket_name')
+@click.option('--project', 'project')
+def main(ped_file_path: str, bucket_name: str, project: str):
+    """Pull ped file from GCS and import to SM"""
+
+    fapi = FamilyApi()
+    client = storage.Client()
+    bucket = client.get_bucket(bucket_name)
+    blob = bucket.blob(ped_file_path)
+    ped_file = blob.download_as_bytes()
```

Comment:
Just a note, you can use [AnyPath](https://github.com/drivendataorg/cloudpathlib) to read cloud object contents:

```
path = AnyPath(ped_file_path)
with path.open() as fh:
    contents = fh.read()
```